relay-server/src/actors/processor.rs

```diff
@@ -1029,6 +1029,28 @@ impl EnvelopeProcessorService {
         });
     }
 
+    /// Normalize monitor check-ins and remove invalid ones.
+    #[cfg(feature = "processing")]
+    fn process_check_ins(&self, state: &mut ProcessEnvelopeState) {
+        state.envelope.retain_items(|item| {
+            if item.ty() != &ItemType::CheckIn {
+                return true;
+            }
+
+            match relay_monitors::process_check_in(&item.payload()) {
+                Ok(processed) => {
+                    item.set_payload(ContentType::Json, processed);
+                    true
+                }
+                Err(error) => {
+                    // TODO: Track an outcome.
```

Comment:
Are we OK with not tracking outcomes here?

Reply:
Yes, the backend also doesn't track outcomes yet, and it's not yet clear whether we want to track per check-in update or only per check-in.